### PR TITLE
[e2e] Fix AuthEnable/Disable e2e test implementations

### PR DESCRIPTION
**Pull request by veshij**
etcdctl AuthEnable and AuthDisable subcommands does not support json output and produce single-line output even is json mode is requested.
Existing implementations does not work correctly in newly added e2e test.

Splitting https://github.com/etcd-io/etcd/pull/14574 into multiple PRs.

Signed-off-by: Oleg Guba <oleg@dropbox.com>

### CODE REVIEW

File: tests/framework/e2e/etcdctl.go [hunk 1]
```diff
@@ -466,15 +466,41 @@ func (ctl *EtcdctlV3) AlarmDisarm(ctx context.Context, _ *clientv3.AlarmMember)
...
-	return &resp, err
+	args := []string{"auth", "enable"}
+	cmd, err := SpawnCmd(append(ctl.cmdArgs(), args...), nil)
+	if err != nil {
+		return nil, err
+	}
+	err = cmd.Send(strings.Join(args, " "))
+	if err != nil {
+		return nil, err
+	}
```
Comment (ahrtr) [inline tests/framework/e2e/etcdctl.go:477]:
```suggestion
	args := []string{"auth", "enable"}
	cmd, err := SpawnCmd(ctl.cmdArgs(args...), nil)
	if err != nil {
		return nil, err
	}
```

File: tests/framework/e2e/etcdctl.go [hunk 2]
```diff
@@ -466,15 +466,41 @@ func (ctl *EtcdctlV3) AlarmDisarm(ctx context.Context, _ *clientv3.AlarmMember)
...
-	return &resp, err
+	args := []string{"auth", "disable"}
+	cmd, err := SpawnCmd(append(ctl.cmdArgs(), args...), nil)
+	if err != nil {
+		return nil, err
+	}
+	err = cmd.Send(strings.Join(args, " "))
+	if err != nil {
+		return nil, err
+	}
```
Comment (ahrtr) [inline tests/framework/e2e/etcdctl.go:496]:
```suggestion
	args := []string{"auth", "disable"}
	cmd, err := SpawnCmd(ctl.cmdArgs(args...), nil)
	if err != nil {
		return nil, err
	}
```

**Comment by ahrtr** [comment]
Please signoff the commit.

**Comment by veshij** [comment]
> Please signoff the commit.

done.

**Review by ahrtr — APPROVED** [review]
LGTM

Thank you @veshij

**Pull request merged**